tests/docker/test.sh

```diff
@@ -2,7 +2,7 @@
 
 # MAIN EXECUTION POINT
 cd "$WORKSPACE"
-tox
+tox -e unit-py35-django111 -- tests/unittests/general/web_middleware_test.py::TestEnsureAccount
```

Comment:
Erhm, no.

Reply:
Ahaha! Thinking I should maybe put this file in an extra .gitignore... 'git add -u' is dangerous :)